packages/graphql-fixtures/src/fill.ts

```diff
@@ -18,7 +18,7 @@ import {
   Field,
   Operation,
   InlineFragment,
-} from 'graphql-tool-utilities/ast';
+} from 'graphql-tool-utilities/lib/ast';
```

Comment:
importing from lib seems wrong?

Reply:
I'm not sure how to make this work properly otherwise, since the `ast` module will live in `graphql-tool-utilities/lib/ast` at runtime.